src/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+cet_make_library(
+    SOURCE
+      CrvHitInfoReco.cc
+      SConscript
+      InfoStructHelper.cc
+      CrvInfoHelper.cc
+      classes.h
+      CrvHitInfoMC.cc
+      CrvPlaneInfoMC.cc
+      classes_def.xml
+      CrvPulseInfoReco.cc
+      CrvSummaryMC.cc
+      SurfaceStepInfo.cc
+      InfoMCStructHelper.cc
+    LIBRARIES PUBLIC
+      KinKal_Trajectory
+      
+      Offline::CRVReco
+      Offline::CRVResponse
+      Offline::MCDataProducts
+      Offline::RecoDataProducts
+      Offline::DataProducts
+      Offline::KinKalGeom
+      Offline::Mu2eKinKal
+      Offline::TrackerConditions
+      Offline::Mu2eUtilities
+      Offline::GeneralUtilities
+
+    LIBRARY_NAME eventntuple
+)
+
+art_dictionary( NO_CHECK_CLASS_VERSION # For some reason this segfaults
```

Comment:
Can you clarify this comment? The code segfaults without this ```NO_CHECK_CLASS_VERSION``` or something else?

Reply:
Actually I can't! that comment is from cut-n-paste from other cmake files.  We might find the source or post the question, if you think it is important.  Or I can remove the comment.

Reply:
No worries. I'm sure it's fine. We can always work it out later if we need to